plugins/scaffolder/src/next/TemplateWizardPage/Stepper/Stepper.tsx

```diff
@@ -149,6 +151,19 @@ export const Stepper = (props: StepperProps) => {
             onSubmit={handleNext}
             fields={extensions}
             showErrorList={false}
+            transformErrors={props.transformErrors}
+            // this is needed because handleNext is really triggering the validation
+            // which only happens onSubmit
+            onChange={({ formData, schema }) => {
+              if (props.transformErrors) {
+                validator.validateFormData(
```

Comment:
Does this mean that the ENTIRE form is validated on every change, including fields lower down whose values you haven't yet had the time to enter?

Reply:
@freben I meant to remove that (and I just did).  It is not needed.